packages/eslint-plugin/src/rules/prefer-nullish-coalescing.ts

```diff
@@ -11,6 +12,11 @@ export type Options = [
     ignoreTernaryTests?: boolean;
     ignoreMixedLogicalExpressions?: boolean;
     allowRuleToRunWithoutStrictNullChecksIKnowWhatIAmDoing?: boolean;
+    ignorePrimitives?: {
+      string?: boolean;
+      boolean?: boolean;
+      number?: boolean;
```

Comment:
Nit: could you alphabetize these in all the places they're declared? It's easier to read through docs when the ordering is predictable.

```suggestion
      boolean?: boolean;
      number?: boolean;
      string?: boolean;
```

This also applies to where to place the `ignorePrimitives` option. Feels like it should be after `ignoreMixedLogicalExpression`. _(I realize the options already aren't alphabetical, heh - that's an issue that can be a separate followup)_